### ISSUE TITLE
Data file structure/creation

### ISSUE THREAD

**Issue by ckingdev** [neurosnap/sentences#11]
I'd like to use train a punkt model on a custom corprus- in this case, a large set of tweets collected from the Twitter API. While it is technically english, I'm not having great results with any off the shelf tokenizer available in go. Twitter obviously has some idiosyncrisies- unique abbreviations, the misspellings inherent to web text, urls, emoji...and so on. I wanted to take an unsupervised approach first which led me to punkt and this package.

I've taken a quick look at the data files provided in the repo, but it isn't completely clear what the structure is, or how they were created. I'm happy to make a pull request with my work if I see some results, but if you could point me in the right direction as to the generation of the data file for a custom corprus I'd really appreciate it. It's entirely possible that I'm missing some existing documentation. If not, I'd be happy to clean up any explanation you can give and make a pull request to include it in docs. Thanks!

**Comment by neurosnap** [comment]
Hey!

Thanks for submitting the issue, it's great to see that people are trying to use this library.

The files were actually created from `nltk` punkt tokenizer training data.  At this time this library was meant to be a 1:1 port of the library so I ended up using their training data.

I'll follow up with what the keys actually mean when I can get a chance to review the data structures -- it's been awhile.

If you wanted to use your own training data, we would have to write the nltk trainer, I attempted to do it at one point but found grabbing their data easier.

https://github.com/nltk/nltk/blob/develop/nltk/tokenize/punkt.py
